src/main/java/uk/gov/hmcts/sptribs/common/event/CreateTestCase.java

```diff
@@ -67,6 +68,9 @@ public class CreateTestCase implements CCDConfig<CaseData, State, UserRole> {
     @Autowired
     private SubmissionService submissionService;
 
+    @Autowired
```

Comment:
Don't think supplementary data is needed for case linking feature. Remove them from everywhere in this PR, if not required.

Reply:
Removed supplementary data as its not required for Case Links